pkg/didcomm/packer/anoncrypt/pack.go

```diff
@@ -101,6 +107,26 @@ func (p *Packer) Pack(payload, _ []byte, recipientsPubKeys [][]byte) ([]byte, er
 	return []byte(s), nil
 }
 
+func jweTypesFromOpts(opts []packer.Opts) (string, string) {
+	pOpts := packer.NewOpt()
+
+	for _, opt := range opts {
+		opt(pOpts)
+	}
+
+	typ := pOpts.TypHeader()
+	if typ == "" {
+		typ = packer.EncodingTypeV2
```

Comment:
DIDComm V2 cannot be default while not being part of the core group in AIP 2.0

Reply:
this anoncrypt implementation is not going to be supported by AIP 2.0 since their anoncrypt version is different:
* this implementation uses ECDH-ES (no sender info used at all)
* AIP 2.0's version of anoncypt uses a fresh DID with a fresh sender key (this is technically our AFG's authcrypt implementation with a new random sender key).

But to answer your concern, since JWE packers (both anoncrypt/authcrypt) are currently not used AFG DIDComm, there's no point in making V1 the default, we'll support V2 from the get go when DIDComm is configured to use these packers by default.